introduction.md

```diff
@@ -0,0 +1 @@
+Hi. I'm Doc! It's nice to be part of the Mastering GitHub series!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.